### PR TITLE
Remove unreachable code

### PR DESCRIPTION
**Pull request by DanielRuf**
This PR removes unreachable code.

### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [ ] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) before.
- [ ] Run the shell script(s) under `./bin/` (or Windows batch scripts under`.\bin\windows`) to update Petstore samples related to your fix. This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit, and these must match the expectations made by your contribution. You only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the code or mustache templates for a language (`{LANG}`) (e.g. php, ruby, python, etc).
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `4.3.x`, `5.0.x`. Default: `master`.
- [x] Copy the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### CODE REVIEW

**Comment by DanielRuf** [comment]
@jebentier @dkarlovi @mandrean @jfastnacht @ackintosh @ybelenko @renepardon

File: samples/client/petstore/php/OpenAPIClient-php/lib/Api/DefaultApi.php [hunk 1]
```diff
@@ -156,21 +156,6 @@ public function fooGetWithHttpInfo()
                         $response->getHeaders()
                     ];
             }
```
Comment (renepardon) [inline samples/client/petstore/php/OpenAPIClient-php/lib/Api/DefaultApi.php]: I would also remove the switch block as there is only a default case, no real condition

File: samples/server/petstore/php-slim/lib/Api/AbstractAnotherFakeApi.php [hunk 2]
```diff
@@ -73,7 +73,5 @@ public function call123TestSpecialTags(ServerRequestInterface $request, Response
         $body = $request->getParsedBody();
         $message = "How about implementing call123TestSpecialTags as a PATCH method in OpenAPIServer\Api\AnotherFakeApi class?";
         throw new Exception($message);
```
Comment (renepardon) [inline samples/server/petstore/php-slim/lib/Api/AbstractAnotherFakeApi.php]: Declaration of $message variable is useless if only used once. I would paste the message to the constructor directly

**Comment by wing328** [comment]
You will need to update the mustache templates instead of the PHP samples

Ref: https://github.com/OpenAPITools/openapi-generator/tree/master/modules/openapi-generator/src/main/resources

**Comment by wing328** [comment]
Closing this one. Please file a new one by updating the mustache templates instead. PM me via Slack if you need help.

**Comment by DanielRuf** [comment]
We could have kept that open (I could amend the commit).

**Pull request closed without merging**